c2corg_api/views/user.py

```diff
@@ -116,20 +120,128 @@ def post(self):
             log.warning('Error persisting user', exc_info=True)
             raise HTTPInternalServerError('Error persisting user')
 
-        # also create a version for the profile
-        DocumentRest.create_new_version(user.profile, user.id)
```

Comment:
@tsauerwein I added a commit to restore previous behaviour (a profile and a version is created immediately during registration). We will revisit this later when we handle cleaning of users not validated in the delay.